### ISSUE TITLE
Just wanted to say hi!

### ISSUE THREAD

**Issue by tin2tin** [IceSandwich/IceToolbag#1]
Cool to see you're coding an add-on for the vse. Is it like a modifier system for effects? 
If you run into a dead-end, you can find a lot of (used to be working code) for the vse here: https://blenderartists.org/t/video-sequence-editor-news-add-ons/1188770

If you haven't already, you could also consider to join the vse chat here: https://blender.chat/channel/vse 

Afaik, work was done to improve the performance of animation of vse strips in 3.0, so maybe that will have an impact on your add-on too.

**Comment by IceSandwich** [comment]
Hi, this addon is still in progress. Just like the name, this toolbag need to contain lots of things. About its function I will finish the document later.
As for blender 3.0, i don't know what to do right now because this addon has been coded quite a lot. If somebody are interested in it, maybe i will continually improve it.

**Comment by tin2tin** [comment]
I don't think there are any api breaking changes between 2.93 and 3.0 so what you will experience is mainly be a speed increase of processing animated strips. 

Are you aiming at making a modifier stack option for the effect strips and sidebar properties?

**Comment by IceSandwich** [comment]
Yes i  want to make a modifier stack like Premiere does. Using the combination of effect strips provided from blender can result in another effect.
